test_psa_target.py

```diff
@@ -168,6 +184,16 @@ def _execute_test(args, suite):
     :param args: Command-line arguments
     :param suite: Test suite
     """
+
+    # Issue : https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/issues/49
+    # There is no support for this target to run Firmware Framework tests
+    if suite == "IPC" and args.mcu == "ARM_MUSCA_S1":
+        logging.info(
+            "%s config is not supported for %s target" % (suite, args.mcu)
+        )
+        test_results[suite] = "SKIPPED"
+        return
+
```

Comment:
We should avoid building the target too with IPC config. I think it affects as good as execution

Reply:
I have moved this check to `_run_compliance_test` , so it happens before the build.